### DISCUSSION TITLE
Error from not used `bib.bib` file

### DISCUSSION THREAD

**Discussion by friendly** [quarto-dev/quarto-cli#6439] [Q&A] [answered]
### Description

In a book project, I declare multiple `.bib` files in `_quarto.yml` for finding references, 

```
bibliography:
  - bib/references.bib
  - bib/R-refs.bib
  - bib/packages.bib
  - "C:/Dropbox/localtexmf/bibtex/bib/graphics.bib"
  - "C:/Dropbox/localtexmf/bibtex/bib/statistics.bib"
  - "C:/Dropbox/localtexmf/bibtex/bib/timeref.bib"
biblio-style: apalike
link-citations: true
```

and the references there are appropriately found.  Yet, when I render the book, I get an error at the end referring to a
`bib.bib` file quarto seems to be looking for,

```
File bib.bib not found in resource path
ERROR: File bib.bib not found in resource path
```

This causes quarto to halt, and not open the rendered HTML file.

Does anyone have an idea what might be causing this?  
Where in the source code might quarto be creating a requirement for a file named `bib.bib` ?

**Comment by mcanouil** [comment]
I would say the issue is Dropbox.
Move the files from to a place not synced to any services. Do you have the same behaviour?

**Comment by friendly** [comment]
Not Dropbox -- I moved this project out of Dropbox ages ago.

**Reply by friendly** [reply]
But, thanks for looking at this!

**Reply by mcanouil** [reply]
Well the files paths clearly show "Dropbox" in them.

The thing is currently we can't reproduce this, thus investigate.
Could you share a minimal repository highlighting the issue?

**Comment by friendly** [comment]
Oh, that's where I keep some of my `.bib` files, but not the book project.  
I'll first try copying them to my project `bib/` folder to see if that makes a difference,

**Comment by friendly** [comment]
It makes no difference if all my `.bib` files are in the project directory

```
bibliography:
  - bib/references.bib
  - bib/R-refs.bib
  - bib/packages.bib
  - bib/graphics.bib
  - bib/statistics.bib
  - bib/timeref.bib
biblio-style: apalike
link-citations: true
 ...
```

Is there anything like a **debug** mode / option in quarto?
Is there anything like a switch to turn ERRORs into WARNINGs so as not to quit?
(If not, could I suggest something like these as **new features**?)

**Reply by mcanouil** [reply]
See <https://quarto.org/docs/troubleshooting/index.html> for advanced options to debug.

**Comment by dragonstyle** [comment]
I just tried a few different specifications of bibliography files on Windows and MacOS and wasn't able to reproduce any issues. Do you have a complete example that reproduces the example that you could share (ideally a minimal version)?

**Comment by friendly** [comment, marked as answer]
I copied my book project to a different computer and tried to compile there.  I don't get this error, so it must be some problem with the configuration on my main machine. I should have tried this earlier.

Thanks though for the replies.  **Verbose mode** is quite handy.
